web/src/p2k16/web/static/p2k16-header.html

```diff
@@ -13,6 +13,7 @@
       <li ng-class="{'front-page': 'active'}[active]"><a href="#">Home</a></li>
       <li ng-class="{'badges': 'active'}[active]"><a href="#!badges">Badges</a></li>
       <li ng-class="{'tool': 'active'}[active]"><a href="#!tool">Tools</a></li>
+      <li><a ng-click="header.manageBilling()">Membership</a></li>
```

Comment:
Should it have been in the dropdown near "My profile" and "My public page" rather than in the main menu? I can't remember where it used to be

Reply:
Yeah it was. My idea was to make it a bit more prominent. Feel free to change it.